ntpStuff.ino

```diff
@@ -76,8 +76,8 @@ time_t getNtpTime()
     TelnetStream.println(ntpServerIP);
     TelnetStream.flush();
     sendNTPpacket(ntpServerIP);
-    uint32_t beginWait = millis();
-    while (millis() - beginWait < 1500) 
+    DECLARE_TIMER_MS(beginWait, 1500);
```

Comment:
Wat is hier de toegevoegde waarde van de declare timer?
Je moet gewoon 1x maximaal 1500ms wachten of je wat binnen hebt gekregen. In dit geval vind ik de timer niet voor duidelijkheid zorgen.
Eventueel zou je er: 
   while ((int32_t)(millis() - beginWait) < 1500)
van kunnen maken..